# Proteomics
This repository serves as the place to store information and projects related to proteomics from the proteomics course at NYU.


## Proteomics Databases:
Below is a summary of Proteomics Databases that were explroed in this course.
### Posttranslational Modification Database 
*'Phosida allows the retrieval of phosphorylation, acetylation, and N-glycosylation data of any protein of interest. It lists posttranslational modification sites associated with particular projects and proteomes or, alternatively, displays posttranslational modifications found for any protein or protein group of interest. In addition, structural and evolutionary information on each modified protein and posttranslational modification site is integrated.'*
http://phosida.de

### Expasy
*'It is an extensible and integrative portal which provides access to over 160 databases and software tools, developed by SIB Groups and supporting a range of life science and clinical research domains, from genomics, proteomics and structural biology, to evolution and phylogeny, systems biology and medical chemistry.'*
https://www.expasy.org/about

Part of Expasy is https://world-2dpage.expasy.org/list/ and https://world-2dpage.expasy.org/swiss-2dpage/ where analysis of 2D PAGE Gels can be ssisted by a database of gels. These databases allow for exploration of proteome gels in different types of organisms. 


### HPRD
Human Protein Reference Database. Integrates Gene Ontology data, sequence data, naming information, domains and motifs, interactions and other information on protein queries of interest. Also provides pictorial representation of the organization of the protein and post-translational modifications. *PTM & Substrates* tab is helpful since it summarizes the information on prescence of different postranslational modifications as residue and type of modification-also mentions enzymes invoved. HPRD also provides Swiss-Prot ID which is helpful to cross check UniProt information on the protein of interest. 
https://www.hprd.org

### Pfam (now hosted on interpro)
Collection of conserved sequences identified by hidden markov models. 
https://www.ebi.ac.uk/interpro/

### Protein Data Bank (PDB)
Contains information on protein structure found by NMR, Cryo-EM or X-Ray Crystallography. Any structure that has been resolved will be usually deposited in this databank. 
https://www.rcsb.org

### OMIM
An Online Catalog of Human Genes and Genetic Disorders. OMIM also contains descriptions that have been extracted from the literature. Heavily relies on language processing to extract information from literature. 
https://www.omim.org

### Phosphosite
*'PhosphoSitePlusÂ® provides comprehensive information and tools for the study of protein post-translational modifications (PTMs) including phosphorylation, acetylation, and more.'*
https://www.phosphosite.org/homeAction.action

### NetPath
Records signal transduction pathways
http://www.netpath.org

### Cytoscape
*'software platform for visualizing complex networks and integrating these with any type of attribute data'*
https://cytoscape.org

### IntAct
Provides analysis tools for molecular interaction data.
https://www.ebi.ac.uk/intact/home

## DiP
*'The DIPTM database catalogs experimentally determined interactions between proteins. It combines information from a variety of sources to create a single, consistent set of protein-protein interactions. '*
https://dip.doe-mbi.ucla.edu/dip/Main.cgi

### KEGG
Kyoto Encyclopedia of Genes and Genomes. Contains signaling pathway information divided by different types of processes. 

### Reactome
Pathway-related database. Allows you to find reactions proteins and pathways and their relationships. 
https://reactome.org

###  GPM
*'The Generalized Proteoform Meta-analysis Database was constructed to utilize the information obtained by GPM servers to aid in the difficult process of validating peptide MS/MS spectra as well as protein coverage patterns. This database has been integrated into the GPM server pages, allowing users to quickly compare their experimental results with the best results that have been previously observed by users of the machines.'*
https://www.thegpm.org/GPMDB/index.html
